app/code/Magento/Theme/Model/Theme.php

```diff
@@ -79,6 +79,11 @@ class Theme extends \Magento\Framework\Model\AbstractModel implements ThemeInter
      */
     protected $_customFactory;
 
+    /**
+     * @var \Magento\Store\Model\App\Emulation
+     */
+    protected $appEmulation;
```

Comment:
According to [Magento technical guidelines](https://devdocs.magento.com/guides/v2.2/coding-standards/technical-guidelines.html) this property should be private